### PR TITLE
Readability.parse should expose lang attribute of original document.

### PR DESCRIPTION
**Pull request by micha154**

### CODE REVIEW

File: Readability.js [hunk 1]
```diff
@@ -896,6 +896,14 @@ Readability.prototype = {
       let shouldRemoveTitleHeader = true;
 
       while (node) {
+
+        if(node.tagName === "HTML") {
```
Comment (tigleym) [inline Readability.js:900]: This should have a whitespace between the `if` and opening  bracket. This is also causing the tests to fail: https://community-tc.services.mozilla.com/tasks/groups/Lqf0BNjvQoSSmbvA9CS_AA 

Once that's addressed you should run `npm run lint && npm test` to make sure we don't have any other linting/test failures. See contributing guide [here](https://github.com/mozilla/readability/blob/master/CONTRIBUTING.md#tests) for more info.

**Review by tigleym — CHANGES_REQUESTED** [review]

File: Readability.js [hunk 2]
```diff
@@ -896,6 +896,14 @@ Readability.prototype = {
...
+          var articleLang = node.getAttribute("lang");
+          if(articleLang) {
+            this._articleLang = articleLang;
```
Comment (tigleym) [inline Readability.js:903]: I don't think it's necessary for us to have this check here, actually. We could just get the "lang" attribute  and directly assign it.

```
this._articleLang = node.getAttribute("lang");
```

Comment (micha154) [reply Readability.js:903]: if (node.tagName === "HTML") {
          var articleLang = node.getAttribute("lang");
          this._articleLang = articleLang;
        }
I just want to make sure I understand, like this? And then would I run "npm run lint && npm test" in readability not mozilla-unified, correct?

Comment (tigleym) [reply Readability.js:903]: Yeah! Although I think we can (and this is more of a nitpick)  reduce this by one more line:

```
    if (node.tagName === "HTML") {
      this._articleLang = node.getAttribute("lang");
    }
```

And you're correct. You should run `npm run lint && npm test` from the readability repo.

File: Readability.js [hunk 3]
```diff
@@ -2263,6 +2268,7 @@ Readability.prototype = {
       title: this._articleTitle,
       byline: metadata.byline || this._articleByline,
       dir: this._articleDir,
+      lang: this._articleLang,
```
Comment (tigleym) [inline Readability.js:2271]: Could we make sure we have a test for this? These are the steps you should take for this:

1. Assert that Readability.parse() extracts the lang attribute. You should add this after this line https://github.com/mozilla/readability/blob/master/test/test-readability.js#L178. 
2. Generate a test case where the original document has a "lang" attribute on the root. You can generate this test case using the following:

```
node test/generate-testcase.js nytimes-5 https://www.nytimes.com/es/
```

Once you run that command you can manually check the expected-metadata.json file under the generated directory. If you used the above example then the "lang" property will have an "es" value.

3. Make sure to run `npm test && npm run lint`

**Review by tigleym — CHANGES_REQUESTED** [review]
Awesome! We're really close. I think once we have a test case for this we can get this merged :) 

I added some steps you can take to get this test. You can also refer to [this](https://github.com/mozilla/readability/blob/master/CONTRIBUTING.md#add-new-tests) for additional documentation.

File: test/test-readability.js [hunk 4]
```diff
@@ -176,6 +176,10 @@ function runTestsWithItems(label, domGenerationFn, source, expectedContent, expe
     expectedMetadata.dir && it("should extract expected direction", function() {
       expect(result.dir).eql(expectedMetadata.dir);
     });
+
+    expectedMetadata.lang && it("should extract expected direction", function() {
```
Comment (tigleym) [inline test/test-readability.js:180]: We should update the message here to "should extract expected lang"

**Review by tigleym — APPROVED** [review]
Awesome! This looks good. R+ from me after addressing the assertion message

File: test/test-readability.js (hunk 4 above)
Comment (micha154) [reply test/test-readability.js]: Sorry, I don't know how I missed that, should be good now! :)

**Pull request merged**